internal/api/grpc/management/project_converter.go

```diff
@@ -56,6 +58,17 @@ func privateLabelingSettingToDomain(setting proj_pb.PrivateLabelingSetting) doma
 	}
 }
 
+func projectCreateAdminsToCommand(requestAdmins []*mgmt_pb.AddProjectRequest_Admin) []*command.AddProjectAdmin {
+	admins := make([]*command.AddProjectAdmin, len(requestAdmins))
```

Comment:
Can you add a `if len(requestAdmins) == 0` condition and return with nil?
That way we don't have an empty list as attribute.